freqtrade/exchange/exchange.py

```diff
@@ -227,6 +227,12 @@ def precisionMode(self) -> str:
         """exchange ccxt precisionMode"""
         return self._api.precisionMode
 
+    def ccxt_implementation_check(self, method_name: str):
+        api_method = getattr(self._api, method_name, None)
+        if not callable(api_method):
+            raise OperationalException(
+                f"{api_method}() has not been implemented on ccxt.{self.name}")
+
```

Comment:
We have a check for this already with `has`.

ccxt adds all "supported" methods to the "has" dict - which makes it easily queryable for this.
example for binance: 

https://github.com/ccxt/ccxt/blob/477cdce396643f1259d644cb25d498a47c55b9cd/js/binance.js#L33

Asside from that - (i seem to have mentioned that a few times already) - we'll need to check for this at startup - not when it's needed.
Assuming we start trading on a nonsupported exchange - we allow the user to open a position - and then crash the bot because we can't calculate fees.

That's 
a) not good user experience
b) putting the users funds to unnecessary risks

Therefore, please start putting the "startup-checks" in place (look at the bottom of the exchanges's `__init__` page for validations that are already in place.
That's best done alongside adding required methods for modes - as then you'll still remember/know what'll be necessary for "this mode" to work. Otherwise we'll risk to miss certain parts - which will then lead to the above bad UX.